src/Cart/Checkout/AddressManagement/AddAddressModel/AddAddressModel.js

```diff
@@ -0,0 +1,116 @@
+import "./AddAddressModel.css";
+import { useState } from "react";
+import { Button } from "forkui-lib";
+import { IndianStates } from "./StateDatabases/IndianStates";
+import { USStates } from "./StateDatabases/USStates";
+import { AustralianStates } from "./StateDatabases/AustralianStates";
+import { Link } from "react-router-dom";
+export const AddAddressModel = ({
+  setShowAddAddressModel,
+  addresses,
+  setAddresses
+}) => {
+  const [country, setCountry] = useState("India");
+  const [state, setState] = useState("");
+  const [name, setName] = useState("");
+  const [address, setAddress] = useState("");
+  const [city, setCity] = useState("");
+  const [zip, setZip] = useState(0);
+  const [mob, setMob] = useState(0);
+  const updateState = (stateSelected) => {
+    setState(stateSelected);
+  };
+  const submitNewAddress = () => {
+    console.log([
```

Comment:
you can remove the console.log() statement

Reply:
Missed this one. Done now!